server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/Invocation.scala

```diff
@@ -49,8 +54,26 @@ sealed trait ErrorCode {
 
 object ErrorCode {
   case object InvalidArguments extends ErrorCode {
-    override def code: String = "invalidArguments"
+    override def code: String = "error"
 
     override def defaultDescription: String = "One of the arguments is of the wrong type or otherwise invalid, or a required argument is missing."
   }
+
+  case object ServerFail extends ErrorCode {
+    override def code: String = "serverFail"
+
+    override def defaultDescription: String = null
```

Comment:
That is sad, in my opinion, that we don't provide a defaut description here ;-)

Reply:
There probably should be one for each case object extending `ErrorCode` I think :)

Reply:
we provided it where we call it so i made a propose commit.